### ISSUE TITLE
Looks like 2 fps when it's 200 

### ISSUE THREAD

**Issue by Okforplay** [asiekierka/FoamFix#177]
The game isn't slow, when a walk it becames VERY bugged, the frames like stop, and go, looks like it has low fps, but the fps is very high, is this a intentional feature? I've tested with and without my main mods, and it still does it. FoamFix let's me play on the 1.12 but this frame thing is driving me nuts!

**Comment by asiekierka** [comment]
:thinking: 

That sounds pretty strange. Does it happen without FoamFix?

**Comment by Niko-sk2x** [comment]
That looks very much like the game is almost running out of memory. There really isn't anything else that could cause it. Maybe some other bad mod, but it's very unlikely.

**Comment by Okforplay** [comment]
It happen only with foamFix.

**Comment by Okforplay** [comment]
It can be related to low memory too, i've only 4gigs, with 3 allocated to mc

**Comment by Okforplay** [comment]
Without Foamfix, the game runs smooth, but then sttuter and stop, in a cycle, with foamfix the game is very choppy but without those stutters and stops.

**Comment by asiekierka** [comment]
Hmm. But is it the latest version of FoamFix?

**Comment by Okforplay** [comment]
Yes, the 0.10.2. I've tried the older too and the same problem happenned

**Comment by Okforplay** [comment]
I've a GTX 660, i5 third gen, 4 gigs of ram, if that helps

**Comment by asiekierka** [comment]
Perhaps try disabling features until you find one which affects it?

**Comment by Okforplay** [comment]
I've tried some on the config, i'll try everything when i come home

**Comment by angrynerd** [comment]
If you only have 4 gigs of ram you shpuldnt allocate more than 2 and depending on your background tasks no more than 1.
